runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsPlugin.java

```diff
@@ -41,12 +41,7 @@
 import lombok.AccessLevel;
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
-import net.runelite.api.Client;
-import net.runelite.api.GameState;
-import net.runelite.api.GraphicID;
-import net.runelite.api.GraphicsObject;
-import net.runelite.api.MenuAction;
-import net.runelite.api.NPC;
+import net.runelite.api.*;
```

Comment:
Use more specific imports here.

Reply:
Except you're making it less specific. Lol.

Reply:
Nevermind, I was on phone and I didn't realize this was my commit. I couldn't recall this change but I just found out IntelliJ turns imports into wildcard automatically once they reach a certain number (I know because after adding all the imports specifically it removed them all and turned them into a wildcard again). I pushed the change.
There are 16 specific imports now, is it really bad practice to import wildcard in this case? Should wildcard never be used?

Reply:
There isn't any problem with using it in this project, but it's a good practice not to use wildcard imports in general.

A brief article on why it's better to use explicit imports: http://www.javadude.com/articles/importondemandisevil.html